lib/text-formatters.js

```diff
@@ -1,27 +1,30 @@
 /**
  * Commonly-used functions for formatting text in badge labels. Includes
- * ordinal numbers, currency codes, star ratings, etc.
+ * ordinal numbers, star ratings, etc.
  */
+
 'use strict';
 
+const assert = require('assert');
+
+function versionString(version) {
+  const prefix = /^[0-9]/.test(version) ? 'v' : '';
+  return prefix + version;
+}
+exports.versionString = versionString;
+
 function starRating(rating) {
-  var stars = '';
-  while (stars.length < rating) { stars += '★'; }
-  while (stars.length < 5) { stars += '☆'; }
-  return stars;
+  assert(rating >= 0);
```

Comment:
An AssertionError is thrown with a default message. No one should invoke this method with negative number. An exception signals function caller a mistake. It's fail-fast vs return garbage (see prev. version).